text/0035-schema-applies-to-unspecified.md

```diff
@@ -0,0 +1,90 @@
+# Change the Semantics `appliesTo` Fields in the Schema
+
+## Related issues and PRs
+
+- Reference Issues: [cedar#351](https://github.com/cedar-policy/cedar/issues/351)
+- Implementation PR(s):
+
+## Timeline
+
+- Started: 2023-10-25
+
+## Summary
+
+| Scenario | Status Quo | Proposal |
+| --------- | -------- | ------- |
+| Specified `appliesTo` but omitted `principalTypes`/`resourceTypes` | Authz request with this action can only have unspecified entity as principal/resource | Same as status quo |
+| Omitted `appliesTo` | Authz request with this action can only have unspecified entity as principal/resource | Action cannot be used in authz request |
+| `appliesTo.principalTypes: []` / `appliesTo.resourceTypes: []` | Action cannot be used in authz request | ❌ Not allowed |
+| `appliesTo` is defined but all sub-elements are omitted | Authz request with this action can only have unspecified entity as principal/resource; context is an empty record | ❌ Not allowed |
+
+## Motivation
+
+### Current behavior
+
+In Cedar 2.x, if an action's `appliesTo.principalTypes` or `appliesTo.resourceTypes` is not given (or if the entire `appliesTo` element is not given), then it's interpreted as an action that applies to only unspecified principals and resources.
```

Comment:
I don't believe this is correct. If you omit `appliesTo.principalTypes` then just the `principal` component is unspecified. You can still specify `appliesTo.resourceTypes` to require a specific `resource` entity type in a request.

The table, above, also seems to make this mistake.

Reply:
Revised text. Lmk what you think